util/opentelemetry-util-http/src/opentelemetry/util/http/__init__.py

```diff
@@ -84,9 +84,9 @@ def sanitize_header_value(self, header: str, value: str) -> str:
         )
 
     def sanitize_header_values(
-        self, headers: dict, header_regexes: list, normalize_function: callable
-    ) -> dict:
-        values = {}
+        self, headers: dict[str, str], header_regexes: list[str], normalize_function: Callable[[str], str]
+    ) -> dict[str, str]:
+        values: dict[str, str] = {}
```

Comment:
Happy to revert these, I mainly added them to help myself understand the logic in these functions

Reply:
Thanks for adding annotations, we should definitely keep them. but iirc you need the imports from typing before python 3.9 https://docs.python.org/3/library/typing.html#deprecation-timeline-of-major-features